changelogs/fragments/297-docker-connection-config.yml

```diff
@@ -0,0 +1,4 @@
+bugfixes:
+  - "docker connection plugin - fix option handling to be compatible with ansible-core 2.13 (https://github.com/ansible-collections/community.docker/pull/297, https://github.com/ansible-collections/community.docker/issues/307)."
```

Comment:
Do you still want this and the next entry (and the filename) to point to #297? or should it be updated to #313 ? Guess I've never noticed how backport changelog entries are handled.

Reply:
Good point, I wanted to remove the next entry anyway since I removed these new ways in this PR. I'll probably adjust the URL to this PR for this entry.